homeassistant/components/nest/camera_sdm.py

```diff
@@ -33,18 +37,20 @@ async def async_setup_sdm_entry(
             CameraImageTrait.NAME in device.traits
             or CameraLiveStreamTrait.NAME in device.traits
         ):
-            entities.append(NestCamera(device))
+            entities.append(NestCamera(device, hass.data[DATA_FFMPEG]))
```

Comment:
We are not allowed to access the `hass.data` of other integrations.

Instead, add a `async_get_image(stream, outputT_format)` helper function in `components/ffmpeg/__init__.py` that Nest can call.

Reply:
Updated.  I did not update all other integrations that were doing this that I copied from (e.g. ring) since they don't have tests.